### ISSUE TITLE
error: The method or operation is not implemented./n  at Plugin.AudioRecorder.AudioStream.get_Active () [0x00000] in C:\Repos\libs\Plugin.AudioRecorder\Plugin.AudioRecorder\AudioStream.cs:15 

### ISSUE THREAD

**Issue by swetanandha** [NateRickard/Plugin.AudioRecorder#23]

**Comment by NateRickard** [comment]
It looks like you're trying to use the .NET Standard assembly on its own - you can't do that.  From the readme:

>Please note the PCL/.NET Standard support is via "bait & switch", and this will ONLY work alongside the proper platform-specific DLLs/NuGet packages in place.

This means you must install the NuGet package into your iOS, Android, or UWP project too, in order for it to find and use the correct assembly for that platform.

**Comment by chriswestrick** [comment]
I had the same issue, but was able to figure out what the actual problem was. It worked fine for iOS, but not Android. My Android's target platform was 7.1. You don't appear to have a DLL that covers 7.1. When I added the NuGet package to the Android project, it used the incorrect reference (.Net Standard 2.0):

<Reference Include="Plugin.AudioRecorder">
<HintPath>..\packages\Plugin.AudioRecorder.1.0.1\lib\netstandard2.0\Plugin.AudioRecorder.dll</HintPath>

I had to upgrade my project to target 8.1 before I could get this to work with Android

<Reference Include="Plugin.AudioRecorder">
<HintPath>..\packages\Plugin.AudioRecorder.1.0.1\lib\monoandroid81\Plugin.AudioRecorder.dll</HintPath>
